### ISSUE TITLE
Android Support Library 25.4.0 not work

### ISSUE THREAD

**Issue by jk2K** [YoKeyword/Fragmentation#368]
call
```java
popTo(MyAnswerFragment.class, false, new Runnable() {
	@Override
	public void run() {
		start(AnswerDetailFragment.newInstance(questionId));
	}
});                            
```
, and app crash

error message
```
java.lang.NoSuchFieldError: No instance field mAvailIndices of type Ljava/util/ArrayList; in class Landroid/support/v4/app/FragmentManagerImpl; or its superclasses (declaration of 'android.support.v4.app.FragmentManagerImpl' appears in /data/app/com.demo.app-1/split_lib_dependencies_apk.apk:classes2.dex)
	at android.support.v4.app.FragmentTransactionBugFixHack.reorderIndices(FragmentTransactionBugFixHack.java:16)
	at me.yokeyword.fragmentation.FragmentationDelegate$3.run(FragmentationDelegate.java:598)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5521)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:962)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)                                                                 
```

Android Support Library 25.3.1 work well, 

Version info
fragmentation: 0.10.7

**Comment by YoKeyword** [comment]
Yeah, google fixed the bug of pop multiple fragments.

So, don't need the hack.

The next version will be fixed.

**Comment by YoKeyword** [comment]
Released v1.0.0,  it has been fixed     🍺 :)
